query-coordinator/src/main/scala/com/socrata/querycoordinator/QueryRewriter.scala

```diff
@@ -560,12 +560,7 @@ class QueryRewriter(analyzer: SoQLAnalyzer[SoQLType]) extends CompoundQueryRewri
         else h
       }
 
-      val joins = {
-        // might want to tackle updated-ness issue of related table
-        // before we open up join to the public
-        if (!debug && r.joins.nonEmpty) None
-        else rewriteJoin(q.joins, r)
-      }
+      val joins = rewriteJoin(q.joins, r)
```

Comment:
Do we still want the `r.joins.nonEmpty` guard?

Reply:
I think Nil == Nil will work fine.